CHANGELOG.md

```diff
@@ -16,7 +24,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Add `LokiLogTenantIdMissing` alert to detect dropped log lines due to missing tenant.
-- Document how to create alerts based on logs.
```

Comment:
This should stay there I believe

Reply:
No as I merged it this morning and the release predates this

Reply:
Ok, that's fixing previous PR then